charmander/tests/routes/pizza/test_crud.py

```diff
@@ -19,6 +19,8 @@
 from microcosm_postgres.operations import recreate_all
 
 from charmander.app import create_app
+from charmander.models.order_event_type import OrderEventType
```

Comment:
I actually expected updating the tests to be a bit trickier than they were. If this got more complicated, it might make sense to add a `tests/fixtures.py` module which encapsulates some of this logic so the setup doesn't detract from the tests.